c2corg_ui/static/js/documentediting.js

```diff
@@ -141,39 +141,9 @@ app.DocumentEditingController = function($scope, $element, $attrs,
   this.alerts = appAlerts;
 
   /**
-   * first next step would be 2 by default
-   * @export
-   */
-  this.next_step = 2;
-
-  /**
-   * Current step is 1 by default
-   * @export
-   */
-  this.current_step = 1;
-
-  /**
-   * Waypoint type initialised in the edit.html
-   * @export
-   */
-  this.waypoint_type;
-
-  /**
-   * Previous step is 0 by default
-   * @export
-   */
-  this.previous_step = 0;
-
-  /**
-   * Max possible steps for creation/edition
-   * @export
+   * @type {boolean}
+   * @public
```

Comment:
You should have said that when reviewing Alexe's PR ... :laughing: 

Someone on stackoverflow says:

> The access control annotations @private, @protected, and @public are directives for the Closure Compiler that help developers enforce the desired level of visibility for properties and functions.

But if you think it is needed, I can change them back to @export.